src/content/governance/index.md

```diff
@@ -0,0 +1,159 @@
+
+- **Protocol Developers** (a.k.a. "Core Developers" ): these people maintain the various Ethereum implementations (e.g. go-ethereum, Nethermind, Besu, Erigon at the execution layer or Prysm, Lighthouse, Nimbus, Teku, Lodestar at the consensus layer). [More on Ethereum clients](/developers/docs/nodes-and-clients/).
+- **EIP Authors**: these people propose changes to the Ethereum protocol, in the form of Ethereum Improvement Proposals (EIPs). [More on EIPs](/eips/).
+- **Miners/Validators**: these people run nodes that can add new blocks to the Ethereum blockchain.
+- **Node Operators**: these people run nodes that propagate blocks and transactions, rejecting any invalid transaction or block that they come across. [More on nodes](/developers/docs/nodes-and-clients/).
+- **Application/Tooling Developers**: these people write applications that are run on the Ethereum blockchain (e.g. DeFi, NFTs, etc.) or build tooling to interact with Ethereum (e.g. wallets, test suites, etc.). [More on dapps](/dapps/).
+- **Application Users**: these people interact with applications on the Ethereum blockchain.
+- **Ether holders**: these people own the ETH token. [More on ETH](/eth/).
+
+_Note: any individual can be part of multiple of these groups (e.g. a protocol developer could champion an EIP, and run a beacon chain validator, and use DeFi applications). For conceptual clarity, it is easiest to distinguish between them, though._
+
+<Divider />
+
+## What is an EIP? {#what-is-an-eip}
+
+One important process used in Ethereum governance is the proposal of **Ethereum Improvement Proposals (EIPs)**. EIPs are standards specifying potential new features or processes for Ethereum. Anyone within the Ethereum community can create an EIP. For example, none of the authors of EIP-721, the EIP that standardized NFTs, have worked directly on Ethereum's protocol development.
+
+<ButtonLink to="/eips/">More on EIPs</ButtonLink>
+
+<Divider />
+
+## The Formal Process {#formal-process}
+
+The formal process for introducing changes to the Ethereum protocol is as follows:
+
+1. **Propose a Core EIP**: as described in [EIP-1](https://eips.ethereum.org/EIPS/eip-1#core-eips), the first step to formally proposing a change to Ethereum is to detail it in a Core EIP. This will act as the official specification for an EIP that Protocol Developers will implement if accepted.
+
+2. **Present your EIP to Protocol Developers**: once you have a Core EIP, you should get feedback on it from Protocol Developers. You can get feedback can by proposing it for discussion on an [AllCoreDevs call](https://github.com/ethereum/execution-specs/tree/master/network-upgrades#getting-the-considered-for-inclusion-cfi-status). Alternatively, discussion can happen asynchronously on the [Ethereum Magician's forum](https://ethereum-magicians.org/) or in the [Ethereum R&D Discord](https://discord.gg/mncqtgVSVw).
```

Comment:
We should be having people get feedback from the community prior to AllCoreDevs call.  The AllCoreDevs call is a very expensive call, and we want things to show up there only after they are in a very healthy place and have received and applied one or more rounds of feedback.

Reply:
Agreed, we can add another bullet point. Something like:

> 2. **Gather feedback from the community**: Once your EIP has been drafted, you should gather feedback from relevant stakeholders in the community. The amount and variety of feedback will be proportional to the scope and size of your proposed change. Good starting points for this are Ethereum Magicians, ethresear.ch and/or reaching out to projects you think may be directly affected by your EIP.

Reply:
Then, we can modify this bullet: 
```suggestion
3. **Present your EIP to Protocol Developers**: once you have a Core EIP for which you've gathered community input, you should present it to Protocol Developers. You can do so by proposing it for discussion on an [AllCoreDevs call](https://github.com/ethereum/execution-specs/tree/master/network-upgrades#getting-the-considered-for-inclusion-cfi-status). It is likely some discussions will have already happened asynchronously on the [Ethereum Magician's forum](https://ethereum-magicians.org/) or in the [Ethereum R&D Discord](https://discord.gg/mncqtgVSVw).
```